jobs/integr8ly/ocp3/nightly/osd-testing-nightly-trigger.yaml

```diff
@@ -0,0 +1,217 @@
+                                string(name: 'CUSTOMER_ADMIN_USERNAME', value: "customer-admin"),
+                                string(name: 'CUSTOMER_ADMIN_PASSWORD', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+                                string(name: 'NAMESPACE_PREFIX', value: "openshift-"),
+                                string(name: 'WEBAPP_URL', value: "https://tutorial-web-app-openshift-webapp.${ROUTER_SHARD}.${CLUSTER_NAME}.openshiftapps.com"),
+                                booleanParam(name: 'CLEAN_RESOURCES', value: Boolean.valueOf("${CLEAN_RESOURCES}")),
+                                string(name: 'MANIFEST_VERSION', value: "${MANIFEST_VERSION}"),
+                                booleanParam(name: 'TESTING_MASTER', value: Boolean.valueOf("${TESTING_MASTER}")),
+                                string(name: 'SSO_URL', value: "https://sso-openshift-sso.${ROUTER_SHARD}.${CLUSTER_NAME}.openshiftapps.com"),
+                                string(name: 'USERNAME', value: "tester"),
+                                string(name: 'NUMBER_OF_USERS', value: "5"),
+                                string(name: 'TIMEOUT_THRESHOLD', value: "${TIMEOUT_THRESHOLD}")
+                            ]).result
+
+                            if(buildStatus == 'UNSTABLE') {
+                                currentBuild.result = 'UNSTABLE'
+                            } else if(buildStatus == 'SUCCESS') {
+                                currentBuild.result = 'SUCCESS'
+                            } else {
+                                currentBuild.result = 'FAILURE'
+                            }
+                        } // stage
+                    } else {
+                         currentBuild.result = 'UNSTABLE'
+                         println "The test execution was skipped due to unsuccessful login to the cluster!!"
+                    }
+                }// node
+            }}} // timeout, ansiColor, timestamps
+        } catch (caughtError) {
+            currentBuild.result = 'FAILURE'
+            err = caughtError 
+        } finally {
+            notifyBuild(currentBuild.result, err)
+        }
+
+        def notifyBuild(String buildStatus, err) {
+            
+            // In theory, null means success. In practise if there is an error thrown elsewhere than in the triggered jobs (eg. in wokraround scripts)
+            // the job fails but success will be stored in buildStatus anyway
+            buildStatus =  buildStatus ?: 'SUCCESS'
+            
+            String mailRecipients = "${RECIPIENTS}"
+            String subject = "${buildStatus}: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]'"
+            String body = "${subject}\n\n(${env.BUILD_URL})"
+
+            if(err != null) {
+                body = "${body}\n\n FAILURE was ${err}"
+            }
+
+            mail body: body, subject: subject, to: mailRecipients
+        }
+
+        def ocLogin() {
```

Comment:
Very minor, but change name to `ocLoginAsCustomerAdmin()` since username is hard-coded there.